JavaFXUserInterface/src/main/java/controllers/AssetInfoController.java

```diff
@@ -278,6 +278,22 @@ public void attachEvents() {
                 uiUtilities.changeScene(ASSETS_SCENE, assetSaveBtn.getScene());
             }
         });
+
+        // As the window expands or shrinks, attribute panes will adjust to the window size accordingly
+        attributeFlowPane.widthProperty().addListener((obs, oldVal, newVal) -> {
+            double attributeFlowWidth = (double) newVal - 2;
+            if(attributeFlowPane.getChildren().size() < 6) {
+                int nbOfPanes = attributeFlowPane.getChildren().size();
+                attributeFlowWidth = attributeFlowWidth - (247 * nbOfPanes);
+                attributeFlowWidth = attributeFlowWidth / (nbOfPanes - 1);
+                attributeFlowPane.setHgap(attributeFlowWidth);
+            } else {
+                int nbOfPanes = (int) (attributeFlowWidth / 247);
+                attributeFlowWidth = (attributeFlowWidth % 247);
+                attributeFlowWidth = attributeFlowWidth / (nbOfPanes - 1);
+                attributeFlowPane.setHgap(attributeFlowWidth);
```

Comment:
This comment is the same as the one on the other PR. I think it would provide more clarity and better code styling to use static final int with meaningful names instead of just putting integer values for UI element layout or styling. Initializing those values with names will help the user understand why they were used. (Where does this 247 come from)